shared/scss/views/_breakage-form.scss

```diff
@@ -0,0 +1,179 @@
+            display: block;
+            font-weight: bold;
+            font-size: 1em;
+            line-height: 1.35em;
+            margin-bottom: 20px;
+            margin-top: 5px;
+            color: $slate-light;
+        }
+
+        .breakage-form__message {
+            text-align: center;
+            font-weight: normal;
+            color: $slate;
+            padding-bottom: 20px;
+        }
+
+        .breakage-form__success--title {
+            font-size: 1.3em;
+            margin-bottom: 10px;
+        }
+
+        .breakage-form__success--message {
+            font-size: 1em;
+            line-height: 1.45em;
+            font-weight: bold;
+            color: $slate-light;
+        }
+
+        .form__label__select {
+            display: block;
+            text-align: left;
+            font-weight: bold;
+            font-size: 1em;
+            margin-bottom: 8px;
+            color: $slate;
+        }
+
+        .form__select {
+            -webkit-box-sizing: border-box;
+            -moz-box-sizing: border-box;
+            box-sizing: border-box;
+            border-radius: 3px;
+            position: relative;
+            overflow: hidden;
+            line-height: 2.3;
+            height: 2.6em;
+            margin-bottom: 20px;
+            width: 100%;
+            display: block;
+            border: 1px solid $color--dark-silver;
+            background-color: #fafafa;
+            padding: 0;
+            vertical-align: middle;
+            cursor: pointer;
+        }
+
+        .form__select:after {
+            speak: none;
+            font-style: normal;
+            font-weight: normal;
+            text-decoration: none;
+            -webkit-font-smoothing: antialiased;
+            content: "\203A";
+            color: $color--medium-dark-slate;
+            margin-top: -9px;
+            font-size: 20px;
+            line-height: 1;
+            pointer-events: none;
+            vertical-align: middle;
+            display: inline-block;
+            right: 12px;
+            top: 50%;
+            position: absolute;
+            -webkit-transform: rotate(90deg);
+            -moz-transform: rotate(90deg);
+            transform: rotate(90deg);
+        }
+
+        .form__select:hover {
+            background-color: $white;
+        }
+
+        .form__select select {
+            -webkit-appearance: none;
+            -moz-appearance: none;
+            -ms-appearance: none;
+            appearance: none;
+            background: none;
+            color: $slate-light;
+            cursor: pointer;
+            margin: 0;
+            padding: 0 28px 0 1em;
+            position: relative;
+            display: block;
+            font-size: 1em;
+            line-height: inherit;
+            min-width: 10em;
+            width: 100% !important;
+            height: 2.5em !important;
+            outline: none !important;
+            border: none !important
```

Comment:
This is all a hack to make a custom select menu show up. Otherwise it looks really bad on some platforms, like linux.